src/Sylius/Bundle/PromotionBundle/Resources/translations/validators.en.yml

```diff
@@ -15,6 +15,7 @@ sylius:
         invalid_type: Catalog promotion action type is invalid. Please choose a valid type.
         percentage_discount:
             not_in_range: The percentage discount amount must be between 0% and 100%.
+            not_number: The amount of percentage discount must be a number.
```

Comment:
Make these messages consistent, change this one or others for percentage discount amount